projects/project-player-library/src/lib/pages/project-details/project-details.component.ts

```diff
@@ -12,8 +12,10 @@ panelOpenStateForResources = false;
 validationTexts!: string[];
 learningResources: any[] = [];
 ngOnInit(): void {
-  this.getCertificateCriteria();
   this.learningResources = this.projectDetails?.learningResources;
+  setTimeout(()=>{
```

Comment:
Async await didn't work?
Try using ngOnChanges and check

Reply:
implemented the ngOnChange method